OWNERS_ALIASES

```diff
@@ -25,7 +21,6 @@ aliases:
   - mmorhun
   - rcerven
   - tkdchen
-  - stuartwdouglas
   ec-team:
   - simonbaird
   - lcarva
```

Comment:
```suggestion
  - lcarva
  - mbestavros
```

Reply:
@tnevrlka Okay then let's just skip mbestavros, since the bot's complaining

Reply:
I've DM'd him, let's wait for a bit?

Reply:
He has a DevConf presentation so will not be replying soon. Approving this PR, thanks!